### ISSUE TITLE
AudioSession start failed must kill app :(

### ISSUE THREAD

**Issue by davfri** [slavavdovichenko/MediaLibDemos3x#46]
I've a new issue : ERROR: [0x16e733000] 1233: AUIOClient_StartIO failed (-66637)
After several good stream, this error appear and the only choice is kill the app.
Any idea ?

**Comment by slavavdovichenko** [comment]
Is this problem with some demo sample of our kit?
Did it happen with publisher or player?
What version of xcode did you use?

Can you send me the project reproducing this problem (without "lib" folder)?
My email: slavav@themidnightcoders.com

**Comment by davfri** [comment]
Yes the problem appear with the publisher when you use 2 or 3 devices in the same time.
We start and stop many video and after 8 or 12 videos, this error appear and stay.
The only choice is : Kill the app

**Comment by jbeoris** [comment]
Is there any update to this issue? I am having the same problem.
